dotnet/src/IntegrationTests/Agents/MixedAgentTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System;
+using System.ComponentModel;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.Agents.OpenAI;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+using SemanticKernel.IntegrationTests.TestSettings;
+using Xunit;
+
+namespace SemanticKernel.IntegrationTests.Agents;
+
+#pragma warning disable xUnit1004 // Contains test methods used in manual verification. Disable warning for this file only.
+
+public sealed class MixedAgentTests
+{
+    private readonly IConfigurationRoot _configuration = new ConfigurationBuilder()
+            .AddJsonFile(path: "testsettings.json", optional: false, reloadOnChange: true)
+            .AddJsonFile(path: "testsettings.development.json", optional: true, reloadOnChange: true)
+            .AddEnvironmentVariables()
+            .AddUserSecrets<MixedAgentTests>()
+            .Build();
+
+    /// <summary>
+    /// Integration test for <see cref="OpenAIAssistantAgent"/> using function calling
+    /// and targeting Open AI services.
+    /// </summary>
+    [Fact(Skip = "OpenAI will often throttle requests. This test is for manual verification.")]
+    public async Task OpenAIMixedAgentTestAsync()
+    {
+        OpenAIConfiguration openAISettings = this._configuration.GetSection("OpenAI").Get<OpenAIConfiguration>()!;
+        Assert.NotNull(openAISettings);
+
+        await this.ExecuteAgentAsync(
+            this.CreateChatCompletionKernel(openAISettings),
+            new(openAISettings.ApiKey),
+            openAISettings.ModelId);
+    }
+
+    /// <summary>
+    /// Integration test for <see cref="OpenAIAssistantAgent"/> using function calling
+    /// and targeting Azure OpenAI services.
+    /// </summary>
+    [Fact/*(Skip = "No supported endpoint configured.")*/]
```

Comment:
Should we remove comment here?
```suggestion
    [Fact]
```